playground/frontend/playground_components/analysis_options.yaml

```diff
@@ -0,0 +1,4 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
Use this one please: `include: package:total_lints/app.yaml`
By the way, do you feel any lints there are an overkill?

Reply:
I haven't bumped into an obnoxious lint yet.